src/clj/harja/palvelin/palvelut/laadunseuranta/tarkastukset.clj

```diff
@@ -203,6 +203,17 @@
               :x x :y y
               :toleranssi toleranssi)))))
 
+(defn hae-urakan-tieturvallisuusverkko-kartalle [db _user urakka-id]
+  (oikeudet/ei-oikeustarkistusta!)
```

Comment:
Oliko tämä tarkoituksellisesti näin?

Reply:
On, koska sekä urakoiden geometriat että tieturvallisuusverkko on julkista tietoa. Käykö mielestäsi järkeen, vai olisiko parempi kuitenkin rajata joko tarkastusten lukuoikeuteen, tai yleiseen urakan oikeuteen?

Reply:
Joo. Nyt, kun sanot noin, niin menee ehkä enempi tuonne fiilis ja mielipideosastolle ilman varsinaista hyötyä. Mutta mua ei kyllä häiritsisi vaikka tarkastuksen lukuoikeuden tässä vaatisikin.

Reply:
Lisätty lukuoikeuden tarkastus